README.rst

```diff
@@ -77,6 +85,10 @@ This module exports two functions that write CSVs, ``render_to_csv_response`` an
 - ``use_verbose_names`` - (default: ``True``) A boolean determining whether to use the django field's ``verbose_name``, or to use it's regular field name as a column header. Note that if a given field is found in the ``field_header_map``, this value will take precendence.
 - ``field_order`` - (default: ``None``) A list of fields to determine the sort order. This list need not be complete: any fields not specified will follow those in the list with the order they would have otherwise used.
 
+In addition to the above arguments, ``render_to_csv_response`` takes the following optional keyword argument:
+
+- ``streaming`` - (default: ``True``) A boolean determining whether to use ``StreamingHttpResponse`` instead of the normal ``HttpResponse``.
```

Comment:
I think defaulting this to `True` is fine, provided that we bump the major version number in the next release.